packages/website/src/snippets.ts

```diff
@@ -645,42 +627,25 @@ export const voiceCompassSnippet = ({
     environment === Environment.Html ? "nlxai." : ""
   }voiceCompass.create({
   // hard-coded params
-  apiKey: "${
-    // initial eslint integration
-    // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions, @typescript-eslint/prefer-nullish-coalescing
-    config?.apiKey || "REPLACE_WITH_API_KEY"
-  }",
-  workspaceId: "${
-    // initial eslint integration
-    // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions, @typescript-eslint/prefer-nullish-coalescing
-    config?.workspaceId || "REPLACE_WITH_WORKSPACE_ID"
-  }",
-  journeyId: "${
-    // initial eslint integration
-    // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions, @typescript-eslint/prefer-nullish-coalescing
-    config?.journeyId || "REPLACE_WITH_JOURNEY_ID"
-  }",
+  apiKey: "${config?.apiKey ?? "REPLACE_WITH_API_KEY"}",
```

Comment:
these changes cause a regression, this conditional also passes through if the field (`apiKey, `workspaceId`, etc) is an empty string.

https://developers.nlx.ai/multimodal-try-live
![CleanShot 2024-04-18 at 21 53 28@2x](https://github.com/nlxai/sdk/assets/60136/4e05183c-d9a5-40da-a9c5-09129f50ac4a)

vs this branch
![CleanShot 2024-04-18 at 21 53 50@2x](https://github.com/nlxai/sdk/assets/60136/d3f7cd91-add4-4b90-9946-8e5158daa937)